website/src/site/categories/getting-started/designers.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Designers
+eleventyNavigation:
+  parent: Getting Started
+  key: Designers
+  order: 1
+---
+<header class="ds-tokens__main-heading">
+<div class="ds-tokens__heading-wrapper">
+  <h1 class="ds-heading-1">{{title}}</h1>
+  <p class="ds-tokens__heading-description">
+  This guide will walk you through the process of integrating the component library into your Figma workflow, utilizing its components effectively, understanding the component structure, and making the most of the light and dark themes.
+  </p>
+</div>
+</header>
+
+<section class="ds-subpage-section">
+
+<div class="ds-subpage-section__wrapper">
+
+<section>
+
+## Loading the SL Design System Library
```

Comment:
We have a little bit different spacing in Figma than on our website:

<img width="830" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/add6ee65-aba3-40f1-a89d-43f6fd8e50a6">


Do we want to change it in that task, for that subpage only for now or maybe later globally?

Reply:
I know, i discussed with Danielle that we will keep it the same as the spacing and headings (h2 and not h1 as in Figma) as we now have on the component detail page.